guides/v2.2/javascript-dev-guide/widgets/widget_dropdown.md

```diff
@@ -0,0 +1,202 @@
+---
+group: javascript-developer-guide
+subgroup: 3_Widgets
+title: Dropdown widget
+functional_areas: frontend,theme,dropdown
+---
+
+## Overview
+
+Widget source file is [lib/web/mage/dropdowns.js].
+
+The dropdown widget allows to show show on storefront a select box with custom
+content for each of available options.
+
+{:.bs-callout .bs-callout-info} Dropdown widget is not meant to replace default
+HTML select element. By replacing default select in store forms don't expect
+the store to work 100% the same as before changes.
+
+
+**Usages:**
+- [Shipping policy]
+- [Customer menu]
+- [UI Tooltip]
+
+[lib/web/mage/dropdowns.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/mage/dropdowns.js
+[Shipping policy]: {{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Shipping/view/frontend/web/template/checkout/shipping/shipping-policy.html
+[Customer menu]: {{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Customer/view/frontend/templates/account/customer.phtml
+[UI Tooltip]: {{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/frontend/web/templates/form/element/helper/tooltip.html
+
+### HTML markup
+
+```html
+<div class="magento__dropdown-widget">
+    <span class="action toggle" data-toggle="dropdown" aria-haspopup="true">
+        <span>Dropdown Opener Button</span>
+     </span>
+     <ul class="dropdown-options" data-target="dropdown">
+         <li>
+             <span class="item">Dropdown Item One</span>
+         </li>
+         <li>
+             <span class="item"><a href="#">Dropdown Item Two, With Link</a></span>
+         </li>
+         <li>
+             <span class="item">Dropdown Item Three</span>
+         </li>
+     </ul>
+</div>
+```
+
+### Styles
+
+By default the dropdown is not styled. But, there is a handy
+[LESS mixin `.lib-dropdown()`] that comes to the rescue:
```

Comment:
> that comes to the rescue

Please, rephrase this part slightly to have it in more document-style format. Thank you.